src/Nri/Ui/Tabs/V8.elm

```diff
@@ -0,0 +1,473 @@
+
+        { tabList, tabPanels } =
+            TabsInternal.views
+                { focusAndSelect = focusAndSelect
+                , selected = selected
+                , tabs = List.map (\(Tab t) -> t) tabs
+                , tabStyles = tabStyles config.spacing
+                , tabListStyles = stylesTabsAligned config
+                }
+    in
+    Nri.Ui.styled Html.div
+        "Nri-Ui-Tabs__container"
+        []
+        []
+        [ Html.styled Html.div
+            [ Css.displayFlex
+            , Css.alignItems Css.flexEnd
+            , Css.borderBottom (Css.px 1)
+            , Css.borderBottomStyle Css.solid
+            , Css.borderBottomColor Colors.navy
+            , Fonts.baseFont
+            , maybeStyle
+                (\{ topOffset, zIndex, includeMobile } ->
+                    let
+                        styles =
+                            [ Css.position Css.sticky
+                            , Css.top (Css.px topOffset)
+                            , Css.zIndex (Css.int zIndex)
+                            ]
+                    in
+                    if includeMobile then
+                        Css.batch styles
+
+                    else
+                        Css.Media.withMedia [ MediaQuery.notMobile ] styles
+                )
+                config.tabListStickyConfig
+            ]
+            []
+            [ config.title
+                |> Maybe.map viewTitle
+                |> Maybe.withDefault (Html.text "")
+            , tabList
+            ]
+        , tabPanels
+        ]
+
+
+{-| -}
+viewTabDefault : String -> Html msg
+viewTabDefault tabTitle =
+    Html.div
+        [ Attributes.css
+            [ Css.padding4 (Css.px 14) (Css.px 20) (Css.px 12) (Css.px 20)
+            ]
+        ]
+        [ Html.text tabTitle ]
+
+
+viewTitle : String -> Html msg
+viewTitle tabTitle =
+    Html.styled Html.h1
+        [ Css.flexGrow (Css.int 2)
+        , Css.fontSize (Css.px 20)
+        , Css.fontWeight Css.bold
+        , Css.margin4 (Css.px 5) (Css.px 10) (Css.px 10) Css.zero
+        , Css.color Colors.navy
+        ]
+        []
+        [ Html.text tabTitle ]
+
+
+
+-- STYLES
+
+
+stylesTabsAligned : Config -> List Style
+stylesTabsAligned config =
+    let
+        alignmentStyles =
+            case config.alignment of
+                Left ->
+                    Css.justifyContent Css.flexStart
+
+                Center ->
+                    Css.justifyContent Css.center
+
+                Right ->
+                    Css.justifyContent Css.flexEnd
+    in
+    [ alignmentStyles
+    , Css.margin Css.zero
+    , Css.fontSize (Css.px 19)
+    , Css.displayFlex
+    , Css.flexGrow (Css.int 1)
+    , Css.padding Css.zero
+    , maybeStyle Css.backgroundColor config.tabListBackgroundColor
+    , maybeStyle
+        (\color -> MediaQuery.highContrastMode [ Css.backgroundColor color ])
+        config.highContrastTabListBackgroundColor
```

Comment:
So I don't think this is will actually do anything -- I think high contrast mode will coerce the background color no matter what you pass in (unless you set another style that prevents high contrast mode from working, which I don't recommend). 

<img width="566" alt="Screen Shot 2023-03-29 at 9 14 35 AM" src="https://user-images.githubusercontent.com/8811312/228585855-14f63d40-8ed7-422a-8527-1073b4297be9.png">

I think it's probably fine without 🤷‍♀️

Reply:
does it still stop the panel contents from showing through? The point here is for it to be opaque, not necessarily a specific color.

Reply:
Yup!

Reply:
You can test it on assistivlabs (there's a free trial that doesn't require a card or anything)

Reply:
thanks. I couldn't find a case where this made any difference, so I removed it. Looks like the regular BG will be OK for what we're doing 💯